app/src/main/java/com/amaze/filemanager/asynchronous/asynctasks/LoadFilesListTask.java

```diff
@@ -605,24 +605,29 @@ else if (cursor.getCount() > 0 && cursor.moveToFirst()) {
     TrashBin trashBin = AppConfig.getInstance().getTrashBinInstance();
     List<LayoutElementParcelable> deletedFiles = new ArrayList<>();
     if (trashBin != null) {
-      for (TrashBinFile trashBinFile : trashBin.listFilesInBin()) {
-        HybridFile hybridFile =
-            new HybridFile(
-                OpenMode.TRASH_BIN,
-                trashBinFile.getDeletedPath(
-                    AppConfig.getInstance().getTrashBinInstance().getConfig()),
-                trashBinFile.getFileName(),
-                trashBinFile.isDirectory());
-        if (trashBinFile.getDeleteTime() != null) {
-          hybridFile.setLastModified(trashBinFile.getDeleteTime() * 1000);
+
+      List<TrashBinFile> filesInBin = trashBin.listFilesInBin();
```

Comment:
Please do `if ... return;`. That looks much cleaner & less changes.

Reply:
Yes, you are right.